tests/ui/drop/lint-if-let-rescope-with-macro.stderr

```diff
@@ -0,0 +1,39 @@
+error: `if let` assigns a shorter lifetime since Edition 2024
```

Comment:
@jieyouxu This is the test asserting that no machine applicable suggestion is emitted given that the `expr` span cannot be used for suggestion.

Reply:
I wish there is a negative assertion, saying that there is no `NOTE`.

Reply:
(Backlinking from #129460)